static/app/views/settings/organizationDeveloperSettings/index.tsx

```diff
@@ -215,7 +215,7 @@ class OrganizationDeveloperSettings extends AsyncView<Props, State> {
         <SettingsPageHeader
           title={t('Developer Settings')}
           body={
-            <React.Fragment>
```

Comment:
we do it in many other places... it would be cool to use some "script" to replace all the occurrences

Reply:
I don't think we do anymore